yarn-project/circuits.js/src/structs/__snapshots__/function_leaf_preimage.test.ts.snap

```diff
@@ -1,9 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`basic FunctionLeafPreimage serialization serializes a trivial Function Leaf Preimage and prints it 1`] = `
-"function_selector: 0x7b01000000000000000000000000000000000000000000000000000000
-is_private: 0
+"function_selector: 123
+is_private: 1
 vk_hash: 0x0
-acir_hash: 0x31000000082f1200f8e21100000000000000000000
```

Comment:
We're expecting a hash of 0?

Reply:
test does use `Fr.ZERO` as the acir hash

Reply:
kk, unless it breaks things its better to have unique values, but not a big deal